pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
 
    <groupId>ome</groupId>
-   <version>5.2.0</version>
```

Comment:
Generally makes sense and builds are green. Once concern is about the version jump but also the versioning strategy. Are you planning additional work directly on this repository like the addition of classes for the benchmarking? If so, I would be inclined to decouple the repository version from the underlying Bio-Formats version and bump to `5.2.1-SNAPSHOT`